src/assets/css/components/button.css

```diff
@@ -0,0 +1,22 @@
+.btn {
+    color: var(--primary-color);
+    font-family: var(--primary-font);
+    font-size: 16px;
+    font-weight: var(--fw-semibold);
+  }
+  .btn--secondary {
+    padding: 10px 22.22px 9px 25.25px;
```

Comment:
The number is magic. Should check

Reply:
Thank you, On figma I look up the numbers like that, I don't know if I round like this it's still oke?
`padding: 10px 23px 9px 25px;`